integration-tests/README.md

```diff
@@ -68,16 +68,48 @@ can either be 8 or 11.
 Druid's configuration (using Docker) can be overrided by providing -Doverride.config.path=<PATH_TO_FILE>. 
 The file must contain one property per line, the key must start with `druid_` and the format should be snake case. 
 
+## Running tests by docker-compose + mvn
+
+Build druid-cluster, druid-hadoop docker images. From root module run maven command:
+```
+mvn clean install -pl integration-tests -P integration-tests -Ddocker.run.skip=true -Dmaven.test.skip=true
+```
+
+Run druid cluster by docker-compose:
+```
+docker-compose -f integration-tests/docker/docker-compose.yml up
+```
+
+Run integration tests:
+```
+mvn verify -pl integration-tests -P integration-tests -Dgroups=batch-index -Djvm.runtime=8 -Pskip-static-checks -Ddruid.console.skip=true -Dmaven.javadoc.skip=true -Ddocker.run.skip=true -Ddocker.build.skip=true
+```
+
+Run s3 | azure | gcs integration tests:
```

Comment:
I think this is not needed. If we run mvn verify -pl integration-tests -P integration-tests... then it should automatically build and run docker containers. If that is the case, the instruction in Running a Test That Uses Cloud  should be enough

Reply:
removed